docs/swagger.yaml

```
@@ -349,4 +349,30 @@ paths:
       summary: Get all police officers for MUP Admin
       tags:
       - user
+  /user/search:
+    get:
+      description: Performs a fuzzy search for users by first name, last name, or
+        full name with similarity matching
+      parameters:
+      - description: Search query
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)